third-party/linshare/src/test/java/org/apache/james/linshare/LinshareExtension.java

```diff
@@ -109,6 +131,22 @@ public LinshareConfiguration configurationWithJwtFor(LinshareFixture.Credential
             .downloadShare(document.asString(), filename);
     }
 
+    @Override
+    public void beforeAll(ExtensionContext extensionContext){
+        createTechnicalAccount(TechnicalAccountCreationRequest.defaultAccount());
+    }
+
+    private void createTechnicalAccount(TechnicalAccountCreationRequest technicalAccountDTO) {
```

Comment:
technicalAccountDTO is not a `TechnicalAccountDTO`

Reply:
What do you mean that?

Reply:
that the name of your variable doesn't match the object it represents